src/common/metadata-utils/index.ts

```diff
@@ -59,11 +59,12 @@ class MetadataUtils {
       CompactLayout: { sobjectRelated: true },
       CustomField: { sobjectRelated: true, permissionSetTypeName: 'fieldPermissions', permissionSetMemberName: 'field' },
       CustomObject: { sobjectRelated: true, permissionSetTypeName: 'objectPermissions', permissionSetMemberName: 'object' },
+      FieldSet: { sobjectRelated: true },
       ListView: { sobjectRelated: true },
       RecordType: { sobjectRelated: true, permissionSetTypeName: 'recordTypeVisibilities', permissionSetMemberName: 'recordType' },
-      WebLink: { sobjectRelated: true },
+      UserPermission: { sobjectRelated: true, permissionSetTypeName: 'userPermissions', permissionSetMemberName: 'name' },
```

Comment:
are you sure UserPermission is related to a SObject ?

Reply:
Replaced by sobjectRelated: false